PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebuggerRunner.java

```diff
@@ -50,33 +54,56 @@
 import org.apache.http.impl.client.BasicCredentialsProvider;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
-import rx.Single;
+import rx.*;
+import rx.Observable;
 import rx.exceptions.CompositeException;
-import rx.exceptions.Exceptions;
 import rx.schedulers.Schedulers;
 import rx.subjects.PublishSubject;
+import rx.subjects.ReplaySubject;
 
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.net.UnknownServiceException;
 import java.util.AbstractMap.SimpleEntry;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
+import java.util.concurrent.Phaser;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import static rx.exceptions.Exceptions.propagate;
+
 public class SparkBatchJobDebuggerRunner extends GenericDebuggerRunner {
```

Comment:
>SparkBatchJobDebuggerRunner [](start = 13, length = 27)

Should we add a option (like checkbox) in submission UI? Debug executor is only enabled after it is checked and by default set the number of executor to 1?